docs/courses/skill.md

```diff
@@ -0,0 +1,192 @@
+# LibreLingo Documentation on Skill
+
+A **Skill** resides below a module within the course structure. It's the lowest element on the cource hierarchy. It contains data regarding how LibreLingo will ask the questions to the users.
```

Comment:
```suggestion
A **Skill** resides below a Module within the course structure. It's smallest unit in the course hierarchy. It contains data regarding what questions LibreLingo will ask the to the users.
```

Reply:
Fair enough, I think `smallest element` is a good alternative :+1:

Reply:
Are you ok if I keep "lowest element"? I think "lowest" goes well with "element". Sorry I forgot to mention that.

Reply:
ok